### DISCUSSION TITLE
Topics over time per class

### DISCUSSION THREAD

**Discussion by RZanfelice** [MaartenGr/BERTopic#740] [Q&A] [answered]
Hi, first of congrats on the amazing library

I'm looking into topic visualization and found both the topics_over_time and the topics_per_class very helpful, but I would really like to know if there is a way to do both at the same time, for each timestamp I could see the occurrence of each topic per class

**Comment by MaartenGr** [comment, marked as answer]
At the moment, that is not possible. The reasoning for that is that many slices will turn up empty as you typically need lots of data to account for each possible slice. Having said that, I will look into whether it is possible to merge these two functions easily.
